src/Angor/Client/Pages/Signatures.razor

```diff
@@ -602,4 +611,20 @@
 
         public string EventId { get; set; }
     }
+
+    private async void OpenInChat(string NostrPubKey)
+    {
+        var npub = NostrHelper.ConvertHexToNpub(NostrPubKey);
+
+        if (!string.IsNullOrEmpty(npub))
+        {
+            var url = $"https://chat.angor.io/dm/{npub}";
```

Comment:
this is a dependency we can't have that, the chat must be a configurable link in the setting page

Reply:
On the settings page, the user can add and select their chat program, and the default programs have been moved to the configuration file.